src/Ztm.Testing/TransactionData.cs

```diff
@@ -0,0 +1,8 @@
+namespace Ztm.Testing
+{
+    public static class TransactionData
+    {
+        public const string Transaction1 = "01000000016e7a1adf17b1cd472506778d5a21d7d863f18cb43f1b232508fe159982c13fcb000000006a473044022059e66ed5d62ab8be740dbc4b39944e661c71113b7737d1622cc66707c38e8c05022003371ebdd985dc3df3efc3ba4e2bdd876169d9bae338e9b97912b73ecdbac38f012102f972add0df4902428148cae09c2f12ed6241f3302b02cce0e6154538f291299dfeffffff021c446c06000000001976a9149ae4966de9632811e69bc4d31e6d838b470a59e588ac90945502000000001976a914a080fa9739b8e0ac6efcae77bfae7d0135cc3cd288ac00000000";
```

Comment:
Once we renamed the class we should rename this too.

Reply:
Renamed to `Raw1`